scripts/start.sh

```diff
@@ -83,20 +83,20 @@ sed -i.bak -e s/"MINIFICATION: .*"/"$yaml_env_variable"/ app.yaml
 rm app.yaml.bak
 
 # Launch a browser window.
-if [ -f "/usr/bin/google-chrome" ]; then
+if [ $(uname) = "Linux" ]; then
```

Comment:
Note also that we've already defined an ${OS} variable in setup.sh -- might want to reuse that.